examples/Multi_Target/desirability.py

```diff
@@ -116,7 +116,7 @@
     campaign.add_measurements(rec)
 
     print("\n\nInternal measurement dataframe computational representation Y:\n")
-    print(campaign.measurements_targets_comp)
+    print(campaign._measurements_targets_comp)
```

Comment:
Directly accessing a private attribute in one of the examples feels like something we should not do. We should either add a very explicit statement here that this is only for demonstration and not the intended way of accessing stuff or find a different solution.

Reply:
this statement anyway was for debugging and serves no real example-purpose, hence it can go entirely mo